e2e/rdctl.e2e.spec.ts

```diff
@@ -810,7 +816,7 @@ test.describe('Command server', () => {
 
         test('accepts new settings', async() => {
           const oldSettings: Settings = JSON.parse((await rdctl(['list-settings'])).stdout);
-          const body = {
+          const body: any = {
```

Comment:
Please make this a `RecursivePartial<Settings>` or something along those lines to make sure we don't add typos.

Reply:
done